generator/common/beatgen/beatgen.go

```diff
@@ -80,6 +81,13 @@ var configList = []ConfigItem{
 			return "beat"
 		},
 	},
+	{
+		Key:  "beats_revision",
+		Help: "Enter the github.com/elastic/beats revision",
+		Default: func(cfg map[string]string) string {
+			return "master"
```

Comment:
Could we somehow reuse the versions in `libbeat/docs/version.asciidoc` for this default? So if someone creates a new beat from 7.7 branch, 7.7 is used by default and so on.
Maybe we already have some helper in mage to access these versions.